### DISCUSSION TITLE
PROJECT KICKOFF

### DISCUSSION THREAD

**Discussion by charlieikosi** [pinpointltdpng/ODFD-PROJECT#5] [Announcements]
Our project starts off this week starting this month (April). The gantt chart will guide our progress. We shall further refine it to detail other attributes to mark milestones as we progress in the development.

**Comment by anapitalai** [comment]
Week 1
A basic website Layout
header------------Logo(Home) | Contact | About | Login
Body  -------------   Split Screen(depeneding in screen size)       
Footer-------------   Footer

**Reply by anapitalai** [reply]
We can improve that as we progress

**Comment by anapitalai** [comment]
I am trying to digest AR.js https://ar-js-org.github.io/AR.js-Docs/location-based/ using location based AR. I beleive it will fit what we wanna accomplish in terms of AR. Instead of looking at unity and others , we  can use ARjs as a stepping stone to build AR into the web app for the time being forgetting about mobile, we can do that later.

**Reply by anapitalai** [reply]
There is a react template that I use from a react course I did in udemy https://github.com/bradtraversy/proshop_mern.git, we can improve the design later. We need to get the functionalities working.

**Reply by charlieikosi** [reply]
@anapitalai thanks for the link. I'll check it
